vendor/bat-native-ledger/src/bat/ledger/internal/endpoint/promotion/post_claim_gemini/post_claim_gemini.cc

```diff
@@ -86,12 +86,39 @@ type::Result PostClaimGemini::CheckStatusCode(const int status_code) {
   }
 
   if (status_code != net::HTTP_OK) {
+    BLOG(0, "Unexpected HTTP status: " << status_code);
     return type::Result::LEDGER_ERROR;
   }
 
   return type::Result::LEDGER_OK;
 }
 
+type::Result PostClaimGemini::ParseBody(const std::string& body) const {
```

Comment:
It seems like these functions are very similar between the different providers? Do you think it's better to keep them separate so that they can evolve independently? Or would it be worth it to have a common helper for this? (I realize there's probably not a great place to put a common helper at the moment.)

Reply:
I think that `PostClaimBitflyer`, `PostClaimGemini` and `PostClaimUphold` pretty much do the same stuff, so there should be only one linking endpoint client abstraction. 🙂 Obviously, `GeneratePayload` would need to be abstracted away (because of Uphold's dummy transaction verification on the server side), but other than that, they're almost identical.

Reply:
👍

Reply:
Filed https://github.com/brave/brave-browser/issues/19361.